src/gseproject/templates/game/tree.html

```diff
@@ -18,111 +19,222 @@
+/* Custom CSS to adjust the modal position */
+.modal-dialog-centered-at-top {
+    margin-top: 10vh; /* Adjusts the top margin to position the modal closer to the top */
+    display: flex; /* Enables flexbox for this element */
+    justify-content: center; /* Centers the modal horizontally */
+}
+
+/* Ensures the modal is centered horizontally at all times */
+@media (min-width: 576px) {
+    .modal-dialog-centered-at-top {
+        flex-direction: row;
+        justify-content: center;
+    }
 }
 </style>
-  
-<img id="treeImage1" src="{% static 'images/seed.png' %}" alt="Seed">
-<img id="treeImage2" src="{% static 'images/sprout.png' %}" alt="Sprout">
-<img id="treeImage3" src="{% static 'images/sapling.png' %}" alt="Sapling">
-<img id="treeImage4" src="{% static 'images/tree.png' %}" alt="Tree">
-<button id="plantButton" class="btn btn-success">Plant</button>
+<button id="resetButton">Reset</button>
+<img id="close-Button" src="{% static 'images/icons8-close-50.png' %}" alt="Close" onclick="goBack()">
+<div class="start-container">
+    <p id="plantTree-text"> Click the tree to start planting!</p>
+    <img id="plantButton" src="{% static 'images/treebutton.jpg' %}" alt="treebutton" onclick="plantTree()">
+</div>
+<audio id="myAudio" controls>
+    <source src="{% static 'audio/y2mate.is - A Lonely Cherry Tree -D5L36JDKxR0-192k-1710999955.mp3' %}" type="audio/mpeg">
+</audio>
+</div>
+<img id="treeImage1" src="{% static 'images/seed.jpg' %}" alt="Seed">
+<img id="treeImage2" src="{% static 'images/sprout.jpg' %}" alt="Sprout">
+<img id="treeImage3" src="{% static 'images/sapling.jpg' %}" alt="Sapling">
+<img id="treeImage4" src="{% static 'images/tree.jpg' %}" alt="Tree">
+
 
 <div class="countdown">
-  Countdown: <span id="countdownTimer">10</span> seconds
+    <p id="countdownTimer">not started</p> 
 </div>
-       
+<!-- Modal -->
+<div class="modal fade" id="congratulationsModal" tabindex="-1" role="dialog" aria-labelledby="congratulationsModalLabel" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="congratulationsModalLabel">Congratulations!</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          Congratulations! You have grown a full tree and earned 10 bonus cherries, carbon footprint reduced by 1!
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+        </div>
+      </div>
+    </div>
+  </div>
+<!-- Bootstrap CSS -->
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+<!-- Optional JavaScript; choose one of the two! -->
+<!-- Option 1: jQuery and Bootstrap Bundle (includes Popper) -->
```

Comment:
please remove this comment, it makes it obvious where it came from

Reply:
sorted